jubilant/fixtures.py

```diff
@@ -0,0 +1,22 @@
+"""Fixtures for using Jubilant with pytest."""
+
+import secrets
+from collections.abc import Generator
+
+import pytest
+
+from ._juju import Juju
+
+__all__ = ['juju']
+
+
+@pytest.fixture  # TODO: what should the scope be?
+def juju() -> Generator[Juju, None, None]:
+    """Fixture that creates a model and destroys it when done."""
+    juju = Juju()
+    model = 'jubilant-' + secrets.token_hex(6)  # 6 bytes (12 hex digits) should be plenty
+    juju.add_model(model)
+    try:
+        yield juju
+    finally:
+        juju.destroy_model(model, force=True)  # TODO: anything wrong with using --force here?
```

Comment:
Based on my experience with ephemeral juju models that charm integration tests leave behind sometimes:

- force is possibly better than no force
- force is not enough

If storage was created in the model, force doesn't seem to destroy it and then the model can't be removed.

Reply:
I've added `--destroy-storage`.